### ISSUE TITLE
responsive design on Add Work Item

### ISSUE THREAD

**Issue by burrsutter** [openshiftio/openshift.io#1425]
I can not replicate this right now since I do not have an external monitor connected.   
At conferences and meetups, it is common to use an external projector that supports 720p (1280×720) at best.  And what I believe I have seen is that Adding a Work Item in Planner mostly attempts to drop into a 'mobile mode' and the "dialog" fills the entire screen.   1280x720 should still be enough pixels to allow the add work-item screen to work normally.

**Comment by debloper** [comment]
Very reasonable request, and I think it should be escalated to SEV-2 for being significantly bad product experience under certain circumstance (using a popular and standard screen resolution).

**Comment by joshuawilson** [comment]
I think this may actually be a `feature` or `task`. What do you think @michaelkleinhenz ?

**Comment by michaelkleinhenz** [comment]
The breakpoints above indicate that for a 720p resolution, the standard UX is being used. If that is not the case, it is a bug.

Although I just tested this with current prod and it works for me. 720p still uses the standard UX before the first breakpoint. I think what @burrsutter is referring to is the old "select work item type" screen, which was "mobile-like" with unusally large buttons (for *all* resolutions). As we removed this screen and replaced it with the quick-add, this no longer applies.

I'd say we should close this issue.
